.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,35 @@
+---
+name: '🐞 Bug Report'
+about: Create a report to help us improve
+title: ''
+labels: ['type:Bug', 'status:Unconfirmed']
+assignees: ''
+---
+
+<!--
+Found a bug? Please fill out the sections below. 👍
+-->
+
+### Issue Summary
+
+<!--
+A summary of the issue.
+-->
+
+### Steps to Reproduce
+
+1. (for example) Start a new project with `python manage.py runserver`
+2. Edit models.py as follows...
+3. ...
```

Comment:
Would be better if we provide users with a space to write points, instead of editing already present points. The example could be made as a comment.

```suggestion
<!-- For example:
1. (for example) Start a new project with `python manage.py runserver`
2. Edit models.py as follows...
3. ...
-->

1. 
2. 

```